Palindrome/palindrome_string.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+#include <string.h> /*used string library header file for string functions.*/
+main()
+{
+	char str[200], rev[200];
```

Comment:
Use 4 space indentation instead of 1 tab indentation.

Reply:
Done